### DISCUSSION TITLE
Would there be a problem changing the background color of the provider?

### DISCUSSION THREAD

**Discussion by AlexMcComb** [adobe/react-spectrum#2461] [Q&A] [answered]
We are building a plugin with CEP and would like the theme changes to match the CC product background colors when the user changes the CC product theme.  Is there a better way to handle this than adding an `UNSAFE_className` to the `Provider`?:
```
<Provider
      theme={themes[theme]}
      colorScheme={theme.colorScheme}
      UNSAFE_className="provider"
    >
```
When I say background color of the CC Product, I mean this background color:
<img width="252" alt="Screen Shot 2021-10-18 at 11 14 04 AM" src="https://user-images.githubusercontent.com/16233802/137777391-f35c77e4-037a-451b-bf67-9d25b18749a5.png">

**Comment by snowystinger** [comment]
Using the appropriate theme + color scheme is the best way to handle it. If you set it to anything else, you may not meet the minimum contrast ratios required for accessibility

**Comment by cameronmcefee** [comment]
@AlexMcComb Did you ever find a solution for this? While I agree with @snowystinger's logic, the `react-spectrum` themes don't match Illustrators color themes. They're _slightly_ different, and only cover two of the four color stops.

**Reply by AlexMcComb** [reply]
We ditched react-spectrum because of this issue: https://github.com/adobe/react-spectrum/issues/854 and didn't like the workaround and didn't know what other issues we'd run into.  We used an event listener: `window.CSInterface.prototype.addEventListener('com.adobe.csxs.events.ThemeColorChanged', ...)` and created a custom hook to update CSS throughout the app for the colors we needed based on the theme color selected.

**Reply by cameronmcefee** [reply]
👍 Thanks, I was starting to lean that direction myself.

**Comment by AlexMcComb** [comment, marked as answer]
https://github.com/adobe/react-spectrum/discussions/2461#discussioncomment-4151154
